stable/drone/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: drone
 home: https://drone.io/
 icon: https://drone.io/apple-touch-icon.png
-version: 2.0.0-rc.6
-appVersion: 1.0.0-rc.4
+version: 2.0.0-rc.8
+appVersion: 1.0.0-rc.5
```

Comment:
The version of the app is still `1.0.0-rc.4`. Only the chart version should be bumped, you should also merge with the upstream repository to include the changes from the previously merged PR.

Reply:
Merged with the upstream repository, should be good now. The app version seems to be `1.0.0-rc.5` on [Chart.yaml](https://github.com/helm/charts/blob/master/stable/drone/Chart.yamll) currently.

Cheers!